charts/zot/Chart.yaml

```diff
@@ -3,4 +3,4 @@ appVersion: v2.1.0-rc2
 description: A Helm chart for Kubernetes
 name: zot
 type: application
-version: 0.1.56
+version: 0.1.57
```

Comment:
This needs a version bump to 0.1.58
We just released a newer zot version

Reply:
done